.github/workflows/tests.yml

```diff
@@ -10,7 +10,7 @@ jobs:
       fail-fast: false
       matrix:
         os:
-          - macos-latest
+          - macos-11
```

Comment:
[I reported](https://github.com/actions/runner-images/issues/6442) the missing tcl/tk lib symlinks in the macos-12 runner to the github runner images repo.